recipes/xbout/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xbout" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 14a793b000882be6906345e623b5b5da7832cfdb0146cd97a9f53bb0f74df1ff
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36, win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - xarray >=0.16.0
+    - boutdata >=0.1.2
+    - dask[array] >=2.10.0
+    - natsort >=5.5.0
+    - matplotlib >=3.1.1
+    - animatplot >=0.4.1
+    - netcdf4 >=1.4.0
+    - Pillow >=6.1.0
+    - importlib-metadata  # [py<38]
```

Comment:
If making this `noarch: python`, remove the `# [py<38]` selector here and just install `importlib-metadata` unconditionally.

Reply:
It is pure Python, but does not work on Windows (some path-format incompatibility at least - the tests won't pass - and it's not a priority for us to get it working). Actually I originally removed the `noarch: python` because I wanted to skip requiring `importlib-metadata` on Python3.8 - not a big deal but the package is redundant on 3.8 (it's a backport of the 3.8 stdlib's `importlib.metadata`). Is there a benefit to being `noarch`? And can we skip the windows build while being `noarch`?

Reply:
Yes, we can skip the noarch build  on windows

Reply:
> Yes, we can skip the noarch build on windows

@scopatz could you please tell me how? With `skip: true  # [win]` the linter complains
```
`noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.
```
as [expected from the docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages). Is there another way to not have a windows version?

Reply:
It is better to not skip windows, if you can make it noarch

Reply:
As I noted above, the package doesn't work on Windows (at least for the moment). I'd rather not distribute a package that is installable but not usable...

Reply:
That is OK. If you have any non-windows dependencies, it won't actually be installable

Reply:
It's a huge benefit to be noarch because it drastically cuts down on the number of variations of package builds that need to be done (across OS, Python version, and platform architecture).

Reply:
... turns out the easiest solution to this was just to fix the windows bug in our package. Only needed one line changed! I'll make a new PyPi release and update this PR soon.